formats/role.jsonnet

```diff
@@ -49,9 +49,26 @@
         supports_historical_accounts: {
           type: "boolean",
         },
-        seniority :{
+        seniority: {
           type: "integer",
         },
+        whip_organisation: {
+	        type: "object",
```

Comment:
Line 55 looking good now, but still seeing tabs in all the lines below, and you can see the closing `}` on line 71 is too indented as a result.

Reply:
Finally I think it's okay, dear lord 😱